rpcserver.go

```diff
@@ -6566,6 +6566,12 @@ func (r *rpcServer) ExportAllChannelBackups(ctx context.Context,
 func (r *rpcServer) RestoreChannelBackups(ctx context.Context,
 	in *lnrpc.RestoreChanBackupRequest) (*lnrpc.RestoreBackupResponse, error) {
 
+	// The server hasn't yet started, so it won't be able to service any of
+	// our requests, so we'll bail early here.
+	if !r.server.Started() {
+		return nil, ErrServerNotActive
```

Comment:
Similar to this issue https://github.com/lightningnetwork/lnd/issues/5586 -- if we return an actual error code here, we can do a normal type/value check from the PoV of any caller. 

It's a sort of a tangent, but we could also expand this to just live at the `rpcperms` level, and deny all calls (with the same error, other than a set few like GetInfo) that shouldn't be called before the server is fully started up.

Reply:
Cool stuff. Was reading [this](https://jbrandhorst.com/post/grpc-errors/) the other day, and I think it's def better if we could start using the `status`. Speaking of which, since the error codes are global values(from the user's PoV), maybe we should define an error code system if we want to use it in all RPC servers?

Reply:
ha nvm I found this [proposal](https://github.com/lightningnetwork/lnd/pull/5436#issuecomment-909307926).